### PR TITLE
feat(linux): AM62AX, AM62PX: Highlight daughter board support in rele…

### PR DESCRIPTION
**Pull request by Rishikesh-D**
…ase summary

Highlight SK-AM62P + Arducam V3Link (Fusion Mini) and SK-AM62A + DS90UB954-Q1EVM support in the release notes.

### CODE REVIEW

**Review by cshilwant — CHANGES_REQUESTED** [review]
For commit message, checkout the note added in https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#commit-formatting

Since this is a new feature that's eventually highlighted in Release notes, use feat in commit message

File: source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -50,6 +50,7 @@ What's new
   - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
   - RT Kernel : Real-Time Linux Interrupt Latency numbers here - :ref:`RT Interrupt Latencies <RT-linux-performance>`
   - Power Management: DeepSleep, MCU-Only and Partial IO mode - :ref:`Power Management Overview <Power-Management>`
+  - Support for OV5640 + `Arducam V3Link (Fusion Mini) <https://www.arducam.com/product/arducam-v3link-camera-kit-for-ti-development-boards/>`_
```
Comment (devarsht) [inline source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst:53]: same here

Comment (Rishikesh-D) [reply source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst]: @devarsht I have made the changes, please review.

File: source/devices/AM62AX/linux/Release_Specific_Release_Notes.rst [hunk 2]
```diff
@@ -48,6 +48,7 @@ What's new
   - LTS Stable Kernel update to 6.6.58
   - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
   - RT Kernel : Real-Time Linux Interrupt Latency numbers here :ref:`RT Interrupt Latencies <RT-linux-performance>`
+  - Support for OV2312 + `DS90UB954-Q1EVM <https://www.ti.com/tool/DS90UB954-Q1EVM>`_
```
[the remark this answers is not in the document]
Comment (AashvijShenai) [reply source/devices/AM62AX/linux/Release_Specific_Release_Notes.rst]: +1 but do retain the `DS90UB954-Q1EVM <https://www.ti.com/tool/DS90UB954-Q1EVM>`_

**Review by devarsht — APPROVED** [review]

**Review by jeevantelukula — APPROVED** [review]

**Review by jsuhaas22 — APPROVED** [review]

**Pull request merged**